streaming/text/convert/pile.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2022 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""English Wikipedia 2020-01-01 streaming dataset conversion script."""
```

Comment:
Looks like a copy/paste ? need to update for pile dataset ?

Reply:
ah thx